modules/navier_stokes/test/tests/finite_volume/limiters/dispersion-test/tests

```diff
@@ -0,0 +1,130 @@
+[Tests]
+  design = 'INSFVScalarFieldAdvection.md'
+  issues = '#28891'
+  [bottom_left_limited_scalar_advection]
+    requirement = 'The system shall be able to perform a variety of limiting schemes when solving scalar transport equations in cartesian meshes with bottom-left advection. These schemes include'
+    [sou]
+      type = Exodiff
+      input = 'bottom_left_advection.i'
+      exodiff = bottom_left_sou.e
+      cli_args = 'GlobalParams/advected_interp_method=sou Outputs/file_base=bottom_left_sou'
+      rel_err = 1e-4
+      detail = 'second-order upwind'
+      recover = false
```

Comment:
if it diffs in recover we need to know why, or at least have an issue open

Reply:
There is an issue that restart tries to get the file as the other tests are running. This in mostly an issue with the test system rather than this specific test

Reply:
Thanks. Corrected